docs/install/install-cave-beta.md

```diff
@@ -0,0 +1,165 @@
+# Install CAVE - BETA Version
+
+CAVE is the **C**ommon **A**WIPS **V**isualization **E**nvironment that is used for rendering and analyzing data for AWIPS.  The installer may require administrator priviledges to install and may require other system changes (environment variables, etc) as well.
+
+---
+
+## Latest CAVE Versions
+
+- [**Linux: 20.3.2-0.1**](#linux)
+- [**Windows:**](#windows) VM option available, but direct install not yet supported
+- **Mac:** not yet supported
+
+[**View release notes**](https://www.unidata.ucar.edu/blogs/news/tags/awips-release)
+
+!!! note "Version 20.* of CAVE is not compatible with Version 18.* EDEX and vice versa, Version 18.* of CAVE is not compatible with Version 20.* EDEX."
+
+---
+
```

Comment:
I think the "funcationality" section should be moved up here.  I feel like it should be stated right in the begin and that it gets lost near the end.